docs/apm/service-overview.asciidoc

```diff
@@ -14,17 +14,20 @@ high-level visibility into how a service is performing across your infrastructur
 
 [discrete]
 [[service-time-comparison]]
-=== Time series comparison
+=== Time series and expected bounds comparison
 
-Comparing how a service performs relative to a previous time frame can offer additional insight into
-the health of your services. For example, has latency been slowly increasing over time, or did the service
-experience a sudden spike--enabling a time series comparison can provide the answer.
+Comparing how a service performs relative to a previous time frame or to the 
```

Comment:
Not mandatory but this is a long sentence, so I'd personally move the "why bother" to the beginning. For example: 

> For insight into the health of your services, you can compare how a service performs relative to a previous time frame or to the